CHANGES.txt

```diff
@@ -13,3 +13,4 @@ v1.2.7, 2022-07-15 -- Block more user agents - "Petalbot"
 v1.3.0, 2023-02-20 -- Add rate limits
 v1.4.0, 2024-07-26 -- Migrate to Flask.Limiter for rate limits. Breaking Change: 'app' must be passed to 'build_search_view' as the positon 1 argument
 v2.0.0, 2024-08-02 -- Release 1.4.0 as major version change. Migrate to Flask.Limiter for rate limits. Breaking Change: 'app' must be passed to 'build_search_view' as the positon 1 argument
+v2.0.1, 2024-09-10 -- Adds a more granular rate limit: 2/second, 100/minute, 2000/day
```

Comment:
I'm a but confused. In views.py you defined a rate limit of "2000/day;6/minute;1/second" which is different than what you specified here.Also, where is the default value actually specified?

Reply:
['request_limit' is passed to the limiter decorator here](https://github.com/canonical/canonicalwebteam.search/blob/9365dbca5a847631dc7e4766a38007af05aa4398/canonicalwebteam/search/views.py#L72), the rate limit of '"2000/day;6/minute;1/second" is easier QAing, as it is hard to hit what the actual rate limit will be manually eg. "2000/day;100/minute;2/second". There is a note in the 'TODO' section of the description to return this number to the desired value after QA steps have been completed